pkg/config/appsync_server_config.go

```diff
@@ -45,6 +45,7 @@ type AppSyncConfig struct {
 	AppSyncURL         string        `env:"APP_SYNC_URL"`
 	FileSizeLimitBytes int64         `env:"APP_SYNC_SIZE_LIMIT, default=64000"`
 	Timeout            time.Duration `env:"APP_SYNC_TIMEOUT, default=1m"`
+	AppSyncPeriod      time.Duration `env:"APP_SYNC_PERIOD, default=5m"`
```

Comment:
Nit: this is really `AppSyncMinimumPeriod` since it's scheduled externally. I wouldn't want to confuse future readers. Maybe a comment or rename?